src/crimson/os/seastore/transaction_manager.h

```diff
@@ -1450,8 +1453,13 @@ class TransactionManager : public ExtentCallbackInterface {
       direct_partial_off = 0;
       partial_len = direct_length;
     }
-    SUBTRACET(seastore_tm, "getting absent extent from pin {}, 0x{:x}~0x{:x} ...",
-              t, pin, direct_partial_off, partial_len);
+    
+    // are we reading the entire extent?
+    bool is_full_extent = (direct_partial_off == 0 &&
+                           partial_len == direct_length);
```

Comment:
Um, it just came up to me that there might be a problem with this approach. Because, even if `direct_partial_off~partial_len` doesn't cover the whole extent scope, it may still be filling the last hole in the extent, in which case the crc should also be checked. @Matan-B @zhscn @athanatos What do you think?

Reply:
Closing this PR for https://github.com/ceph/ceph/pull/66506.